agent/agent-tooling/src/test/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/ConfigurationTest.java

```diff
@@ -191,6 +192,16 @@ public void shouldParseProcessorConfiguration() throws IOException {
         assertEquals("httpProtocol",attributesExtractConfig.actions.get(0).extractAttribute.groupNames.get(0));
     }
 
+    @Test
+    public void shouldParseAuthenticationConfiguration() throws IOException {
```

Comment:
can you add another test to test all the fields under AadAuthentication? Or update your json test file to cover all.